src/components/ToolBar/DataMenu/SaveToNDExMenuItem.tsx

```diff
@@ -87,6 +91,12 @@ export const SaveToNDExMenuItem = (props: BaseMenuProps): ReactElement => {
       modificationTime: newNdexModificationTime,
     })
 
+    setNetworkModified(currentNetworkId, false)
```

Comment:
When someone uploads the network to NDEx via overwrite, set the network modified status to false.